Source/JavaScriptCore/API/glib/JSCContextInternal.h

```diff
@@ -19,6 +19,6 @@
 
 #pragma once
 
-#include "JSCContext.h"
+#include "jsc/JSCContext.h"
```

Comment:
Why do we need this change?

Reply:
Now that I think of it with a fresher mind, probably there was a `${JavaScriptCoreGLib_DERIVED_SOURCES}/jsc" entry missing in the include paths.

I will be splitting the JSC changes into a separate PR so I will look into this later.